docs/enterprise-api.md

```diff
@@ -797,6 +798,17 @@ curl -d '{
 -H "x-buildbuddy-api-key: YOUR_BUILDBUDDY_API_KEY" \
 -H 'Content-Type: application/json' \
 https://app.buildbuddy.io/api/v1/ExecuteWorkflow
+
+# Execute workflow on commit ABC on branch cool-feature
+curl -d '{
+  "repo_url": "https://github.com/buildbuddy-io/buildbuddy-ci-playground",
+  "ref": "ABC",
```

Comment:
uber nit: make it look a bit more like a commit sha (slightly longer and lowercase)

```suggestion
# Execute workflow on commit abc123 on branch cool-feature
curl -d '{
  "repo_url": "https://github.com/buildbuddy-io/buildbuddy-ci-playground",
  "ref": "abc123",
```